documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -24,7 +24,7 @@ ifeval::["{build}" == "downstream"]
 See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
 endif::[]
 
-. Restart the `CatalogSource` pod:
+. On the *Subscriptions* tab of the *Operator details* page in the {ocp-short} web console, confirm that *Upgrade status* changes from *Up to date* to *Upgrade available*. If it does not, restart the `CatalogSource` pod:
```

Comment:
I think it is better to start with "Confirm that *Upgrade status* changes...."
As the used is already in the right page when execute step 1

And for the same reason 2.a should be removed

Reply:
@mguetta1 If you're already on the *Subscriptions* tab, then that impacts other steps in the procedure as well. Please see my changes.